docs/examples/tls-rustls/src/main.rs

```diff
@@ -0,0 +1,76 @@
+use rustls::pki_types::{CertificateDer, PrivateKeyDer};
```

Comment:
A lot of this code in this example is ported from https://github.com/rustls/tokio-rustls/blob/main/examples/server.rs. I'll give them credit as a source of inspiration before this PR gets merged.